certified-connectors/GroupMgr/apiDefinition.swagger.json

```diff
@@ -1,324 +1,932 @@
+        }
+    ],
+    "consumes": [],
+    "produces": [],
+    "paths": {
+        "/api/webhookrequest/GroupRequested": {
+            "x-ms-notification-content": {
+                "description": "Caught group request.",
+                "schema": {
+                    "$ref": "#/definitions/GroupCompact"
+                }
             },
-            "GroupType": {
-              "type": "string",
-              "description": "The group type.",
-              "title": "Type"
+            "post": {
+                "responses": {
+                    "default": {
+                        "description": "default",
+                        "schema": {}
+                    }
+                },
+                "consumes": [
+                    "application/json"
+                ],
+                "summary": "When a new group is requested. (Approval flow)",
+                "description": "This will trigger when a new group is requested within GroupMgr. (Only groups with this approval flow)",
+                "operationId": "GroupMgrGroupRequested",
+                "x-ms-trigger": "single",
+                "parameters": [{
+                        "name": "body",
+                        "in": "body",
+                        "required": true,
+                        "schema": {
+                            "type": "object",
+                            "properties": {
+                                "name": {
+                                    "type": "string",
+                                    "description": "Name for this approval flow.",
+                                    "title": "Approval name",
+                                    "x-ms-visibility": "important"
+                                },
+                                "webhook": {
+                                    "type": "string",
+                                    "description": "The webhook URL.",
+                                    "title": "Webhook",
+                                    "x-ms-visibility": "internal",
+                                    "x-ms-notification-url": true
+                                }
+                            },
+                            "required": [
+                                "name",
+                                "webhook"
+                            ]
+                        }
+                    }
+                ]
+            }
+        },
+        "/api/webhookrequest/GroupCreated": {
+            "x-ms-notification-content": {
+                "description": "Caught group creation.",
+                "schema": {
+                    "$ref": "#/definitions/GroupExtended"
+                }
             },
-            "Email": {
-              "type": "string",
-              "description": "The groupmail.",
-              "title": "Email"
+            "post": {
+                "responses": {
+                    "default": {
+                        "description": "default",
```

Comment:
@Jwaegebaert  We recommend you to replace all the "default" responses with expected status codes, otherwise users may see errors about incorrect schema instead of real error responses from your service in OpenAPI mode.